test/fuzz_group.cpp

```diff
@@ -678,7 +678,7 @@ void parse_and_apply_instructions(std::string& in, const std::string& path, util
                             bool insert_big_blob = get_next(s) % 2 == 0;
                             if (insert_big_blob) {
                                 size_t rand_char = get_next(s);
-                                size_t blob_size = get_next(s) + ArrayBlob::max_binary_size;
+                                size_t blob_size = get_next(s) + (ArrayBlob::max_binary_size + 1);
```

Comment:
Shouldn't this be `static_cast<uint64_t>(get_int64(s)) % (ArrayBlob::max_binary_size + 1);`? Or better yet, is there a reason for not reusing the existing generator in `construct_mixed()`?

Reply:
Good idea, updated to share the *correct* ™  binary creation code in a function.